style.scss

```diff
@@ -0,0 +1,346 @@
+$cBrown: #bfb2ab;
+
+* {
+    box-sizing: border-box;
+    font-family: 'Oswald', sans-serif;
+    font-family: 'Rasa', serif;
+    font-family: 'Spectral', serif;
+    margin: 0;
+    padding: 0;
+}
+
+
+
+// ========================= mobile ============================================
+@media screen and (max-width:1030px) {
```

Comment:
If you create websites with thinking mobile first, you don't need to define media query for smallest one.
And 1030px is not for mobile. For mobile, you should define it's better to define 320px or 375px. 
```suggestion
@media screen and (max-width:375px) {
```
Otherwise style is collapsed like below.
<img width="1440" alt="スクリーンショット 2021-12-02 23 26 16" src="https://user-images.githubusercontent.com/88401910/144562535-1650606e-1ed0-44cf-97eb-db03629242be.png">
`You` can look at the size of responsive size.